fabfile.py

```diff
@@ -1 +1,32 @@
-from ship_it import fpm
```

Comment:
MAGFest isn't affected by any of these changes, since we don't build Sideboard RPMs or Debian packages and instead deploy via Github (an increasingly common practice).

FWIW, these changes make the packaging more extensible and configurable, which will be nice if/when MAGFest ever starts building and deploying via packages.